docs/fundamentals/single-project.md

```diff
@@ -1,35 +1,195 @@
 ---
 title: ".NET MAUI single project"
-description: "The .NET MAUI single project brings all the platform-specific experiences across Android, iOS, macOS, and Windows, into one shared head project."
-ms.date: 06/10/2021
+description: "Learn about the .NET MAUI single project, which brings all the platform-specific experiences across Android, iOS, macOS, and Windows, into one shared project."
+ms.date: 11/30/2021
 ---
 
 # Single project
 
-.NET Multi-platform App UI (.NET MAUI) single project is a collection of features that brings all the platform-specific experiences you encounter while developing apps into one shared head project that can target Android, iOS, macOS, and Windows.
+.NET Multi-platform App UI (.NET MAUI) single project takes the platform-specific development experiences you typically encounter while developing apps and abstracts them into a single shared project that can target Android, iOS, macOS, and Windows.
 
 [!INCLUDE [docs under construction](~/includes/preview-note.md)]
 
-Single project is enabled using multi-targeting and the use of SDK-style projects in .NET 6. You can still expect access to all the platform-specific experiences and tools when you need them, while enjoying a simplified, shared development experience across all the platforms you're targeting.
+.NET MAUI single project provides a simplified and consistent cross-platform development experience, regardless of the platforms being targeted. .NET MAUI single project provides the following features:
 
-## Simplify development
+- A single shared project that can target Android, iOS, macOS, and Windows.
+- A simplified debug target selection for running your .NET MAUI apps.
+- Shared resource files within the single project.
+- Access to platform-specific APIs and tools when required.
+- A single cross-platform app entry point.
 
-Single project is built on top of a collection of experiences that are being simplified in .NET 6. The following list shows the experiences that will be shared in .NET MAUI single project:
+.NET MAUI single project is enabled using multi-targeting and the use of SDK-style projects in .NET 6.
 
-- Resources
-  - Images
-  - Fonts
-  - App icons
-  - Splash screens
-  - Raw Assets
-- App manifest
-- NuGet
-- Platform-specific code
+## Resource files
 
-All other features are being moved from their own platform-projects into platform folders in the single project.
+Resource management for cross-platform app development has traditionally been problematic. Each platform has its own approach to managing resources, that you have to understand. For example, each platform has differing image requirements that typically involves creating multiple versions of each image at different resolutions. Therefore, a single image typically has to be duplicated multiple times per platform, at different resolutions, with the resulting images having to use different filename and folder conventions on each platform.
```

Comment:
The phrase, `that you have to understand`, maybe could be more general like: `to implement for each platform`. Then it would also not use "you" in the phrase.